src/refract/AppendDecorator.h

```diff
@@ -0,0 +1,75 @@
+//
+//  refract/AppendDecotarod.h
+//  librefract
+//
+//  Created by Jiri Kratochvil on 17/06/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+#ifndef _REFRACT_APPENDDECORATOR_H_
```

Comment:
We don't use `_` before and after these. Please change this to `REFRACT_APPENDDECORATOR_H` and similarly all  the other places you have done this.